github/README.md

```diff
@@ -24,3 +24,8 @@ Working on Issues/Pull Requests
 * Submit a Pull Request with the fix.
 * If your Pull Requests fixes one or more issues, link to them in the discussion.
 * Tag a person (or [team](https://github.com/orgs/subvisual/teams)) if you need their input specifically.
+
+Creating a repository
+-----------------------
+
+* If you are creating a Github repository, use [this](templates) template.
```

Comment:
I know the preview still shows up because the file is called `README`, but the link is wrong. If more templates are added there, this will get confusing